### ISSUE TITLE
[low priority] screen reader audio clip

### ISSUE THREAD

**Issue by slhenry** [w3c/EasyChecks#82]
> The first thing screen readers say when the user goes to a different web page is the page title. Page titles are important for orientation — to help users know where they are and move between open pages.

Would it be useful to have a sound clip of the screen reader going through page titles? Low priority, but maybe neat for people who don't know screen readers?

**Comment by yatil** [comment]
Yes, would be useful. But we should put that in the “How People with Disabilities Use the Web” section and link to there. Learning how the web is used in different circumstances is important and we should not “hide” it inside one of the resources. (we could still have the audio here as well, but having it as an example out of a bigger context would be more useful)
